docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,194 @@
+# Styling Components
+Tizen 6.5 introduces two new methods for styling components derived from the `View` object:
```

Comment:
Could you advice me if overview like the following will be ok ? "Styling of components in NUI is possible in two ways"

Reply:
> Could you advice me if overview like the following will be ok ? "Styling of components in NUI is possible in two ways"

Yes, it is ok. If we add back " derived from the `View` object", the it would be like 
"Styling of components which are derived from the `View` object , is possible in two ways in NUI." or else.

Also, if you have any other information about Styling components in NUI, you can add at this section.

Reply:
thanks, done